Doppler.BillingUser/Infrastructure/BillingRepository.cs

```diff
@@ -828,5 +843,50 @@ [dbo].[BillingCredits] BC
 
             return billingCredit;
         }
+
+        public async Task<PlanDiscountInformation> GetPlanDiscountInformation(int discountId)
+        {
+            using var connection = await _connectionFactory.GetConnection();
+            var discountInformation = await connection.QueryFirstOrDefaultAsync<PlanDiscountInformation>(@"
+SELECT
+    DP.[IdDiscountPlan],
+    DP.[DiscountPlanFee],
+    DP.[MonthPlan],
+    DP.[ApplyPromo]
+FROM
+    [DiscountXPlan] DP
+WHERE
+    DP.[IdDiscountPlan] = @discountId AND DP.[Active] = 1",
+    new { discountId });
+
+            return discountInformation;
+        }
+
+        public async Task UpdateUserSubscriberLimitsAsync(int idUser)
+        {
+            using var connection = await _connectionFactory.GetConnection();
+            using var dtUserCheckLimits = new DataTable();
+            dtUserCheckLimits.Columns.Add(new DataColumn("IdUser", typeof(int)));
+
+            var dataRow = dtUserCheckLimits.NewRow();
+            dataRow["IdUser"] = idUser;
+            dtUserCheckLimits.Rows.Add(dataRow);
+
+            DynamicParameters parameters = new DynamicParameters();
+            parameters.Add("@Table", dtUserCheckLimits.AsTableValuedParameter("TYPEUSERTOCHECKLIMITS"));
+
+            var result = connection.Query("User_UpdateLimits", parameters, commandType: CommandType.StoredProcedure);
+        }
+
+        public async Task<int> ActivateStandBySubscribers(int idUser)
+        {
+            using var connection = await _connectionFactory.GetConnection();
+
+            DynamicParameters parameters = new DynamicParameters();
+            parameters.Add("IdUser", idUser);
```

Comment:
Muy, muy menor, pero creo que podés usar objetos anónimos:

```suggestion
            var parameters = { IdUser = idUser };
```

Reply:
Ahh, ni siquiera estás usando la vriable `parameters`!!!

Por favor, asegurate de que no sobre nada antes de hacer un PR.

Reply:
Entiendo, vi que se podia, pero cuando lo estaba escribiendo me resultó más claro tener los tipos visibles.
Lo modifico.

Reply:
Claro, lo del objeto anónimo es discutible, pero lo que comenté en el segundo comentario es que la variable `parameters` directamente no se usa, eso es un error.